src/app/page.tsx

```diff
@@ -1,7 +1,21 @@
-export default function Home() {
+'use client';
+
+import Form from '@/components/Form';
+import { useRouter } from 'next/navigation';
+import './globals.css';
+import { getBattlePower } from './calc/ScouterCalculator';
+
+export default function RootLayout() {
+  const router = useRouter();
+
+  const handleSubmit = (value: number) => {
+    const followersPower = getBattlePower(value);
+    router.push(`/result?followers=${followersPower}`);
+  };
```

Comment:
[nits]
getBattlePower を経由しているので、followers ではないかなと思います

こうですかね？
```suggestion
  const handleSubmit = (value: number) => {
    const battlePower = getBattlePower(value);
    router.push(`/result?battlePower=${battlePower}`);
  };
```

[imo]
私的にはこっちのが好みかもです
```suggestion
  const handleSubmit = (value: number) => {
    const battlePower = getBattlePower(value);
    router.push(`/result?score=${battlePower}`);
  };
```